### ISSUE TITLE
[Accessibility]: Repo Mangement - Can't navigate to code host connections view 

### ISSUE THREAD

**Issue by jplahn** [sourcegraph/sourcegraph#36419]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33520

### Problem description

From the "Settings" page, you can't navigate to "Code host connections" using only the tab key.

Eventually you tab into the User settings text editor and can't get out of it. 

### Expected behavior

I should be able to navigate to the "Code host connections" menu item on the left hand side of the screen when I open account settings page 

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @jplahn - the "team/repo-management" label was applied to this issue.

**Comment by felixfbecker** [comment]
We removed personal code host connections, so this can't be reproduced anymore.
